### PR TITLE
Dfeusse/update home guides - new index, getting started with MagicBell, guides, etc

### PR DESCRIPTION
**Pull request by dfeusse**
## Change description

> Made changes to the index and getting started with MagicBell pages, and created new guides, for /docs.

## Type of change
- [ ] Bug fix (fixes an issue)
- [x] New feature (adds functionality)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] Application changes have been tested thoroughly
- [ ] Automated tests covering modified code pass

### Security

- [x] Security impact of change has been considered
- [x] Code follows company security practices and guidelines

### Code review 

- [x] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached as necessary
- [x] "Ready for review" label attached and reviewers assigned
- [x] Changes have been reviewed by at least one other contributor
- [x] Pull request linked to task tracker where applicable

### CODE REVIEW

**Comment by dfeusse** [comment]
Screen Shot 2021-09-10 at 8 49 34 AM

File: docs/categories.mdx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+title: Categories
+---
+
+Categories allow notifications to be hrouped together. Why? Soon you will be able to use categories to control which channels notifications are sent to. Users can already take advantage of this. When performing analytics, it is also helpful to group notifications.
```
Comment (josuemontano) [inline docs/categories.mdx:5]: @dfeusse To add more value to this page, could we add an example and explain that you can send any string as a category, too?

File: docs/angular.mdx [hunk 2]
```diff
@@ -0,0 +1,53 @@
+---
+title: Angular.js and MagicBell
+---
+
+### A Complete Walkthrough
```
Comment (josuemontano) [inline docs/angular.mdx:5]: I think you should use h2, instead, since the title is h1.

Comment (unamashana) [reply docs/angular.mdx:5]: @josuemontano That's a good catch. @dfeusse please see https://developers.google.com/search/docs/beginner/seo-starter-guide to learn more about how to structure pages well for SEO/best practices.

**Pull request merged**